### PR TITLE
Add script to install spack build environments for Jenkins

### PR DESCRIPTION
**Pull request by msimberg**
This is what I've been using to update/add Jenkins configurations. The script needs to be run in the `.jenkins/cscs` directory.

### CODE REVIEW

File: .jenkins/cscs/batch.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ source "${src_dir}/.jenkins/cscs/env-common.sh"
 source "${src_dir}/.jenkins/cscs/env-${configuration_name}.sh"
 
 set +e
+spack clean -m
```
Comment (msimberg) [inline .jenkins/cscs/batch.sh:24]: Remove before merging.

Comment (msimberg) [reply .jenkins/cscs/batch.sh]: Removed.

**Review by aurianer — APPROVED** [review]
LGTM thanks!

**Comment by aurianer** [comment]
bors merge

**Comment by msimberg** [comment]
@aurianer, @biddisco note that we might need to add some `chmod`ding to this (or a supporting) script to make it possible for any of us to update/install the instance/packages. Let's update as needed.

**Pull request merged**